plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/IcebergQueryRunner.java

```diff
@@ -429,7 +429,7 @@ public static void main(String[] args)
                     .setInitialTables(TpchTable.getTables())
                     .build();
             log.info("======== SERVER STARTED ========");
-            log.info("\n====\n%s\n====", queryRunner.getCoordinator().getBaseUrl());
+            log.info("\n====\nurl: %s\ndata: %s\n====", queryRunner.getCoordinator().getBaseUrl(), queryRunner.getCoordinator().getBaseDataDir());
```

Comment:
not sure what this is needed for?
why not just query `SHOW CREATE TABLE nation` or something like that to see where the data ends up?

Reply:
SHOW CREATE TABLE (and SCHEMA) returns a relative path from the coordinator data directory. e.g. `local:///tpch/region-fa403d4ceb8542c3a8c42486b59fc86c`

Reply:
good point. i understand the goal is to find data files and be able to inspect them outside of trino
but then, `queryRunner.getCoordinator().getBaseDataDir()` may or may not be the root for the `local:///`. how do you find the files?

Reply:
@dain I vaguely remember that you introduced the `local:///...` testing which came with a speed boost in running the tests ❤️  and also with the confusing part in figuring out where the actual schema & table files corresponding to my tests lie on the local disk storage.

I ended up doing something similar to @ebyhr  in the debugger to retrieve the location of the files.

Do you have a better way at hand to translate from `local:///tpch/region-fa403d4ceb8542c3a8c42486b59fc86c` towards `file:///....` location pointing on the local disk?

Reply:
Updated the code to use `file` file system instead of `local` in query runner's main method. Please take another look.